iso_msgs/msg/Rate.msg

```diff
@@ -0,0 +1,3 @@
+float64 monr;
```

Comment:
Can the naming of this message be more clear? For example message rates. I assume that this message refers to the frequency at which monr, heab and monr2 are sent?

Reply:
I'm trying to replicate the structure of the original [iso22133 structs](https://github.com/RI-SE/iso22133/blob/323bcd51a8f9da2c2a5a4b4eb7b3b7aa3a01ddc9/iso22133.h#L55) as much as possible, but that might not be the way to go. Maybe we need a meeting to clear up how we want to define and use these messages.